{"name": "test_search_add[阿里巴巴]", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: '../config/data.yaml'", "trace": "self = <testcases.test_search.Test_Search object at 0x0000028612209248>\n\n    def setup(self):\n        self.app = App()\n>       self.search = self.app.start().goto_main().goto_market().goto_seacrh()\n\ntestcases\\test_search.py:10: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <page.app.App object at 0x0000028612209788>\n\n    def start(self):\n        #如果driver=None，则说明是首次打开app\n        if self._driver == None:\n            logging.info('driver=None，读取yaml配置文件')\n>           with open('../config/data.yaml','r',encoding='utf-8') as file:\nE           FileNotFoundError: [Errno 2] No such file or directory: '../config/data.yaml'\n\npage\\app.py:17: FileNotFoundError"}, "parameters": [{"name": "name", "value": "'阿里巴巴'"}], "start": 1620122274987, "stop": 1620122274987, "uuid": "8e89f941-b634-48df-8d58-0c08f61bce29", "historyId": "c794dac9ae7ee7ab1487e02c8e5359e0", "testCaseId": "a8c4edba3ebe388ff7ebef19ab7b9b8d", "fullName": "testcases.test_search.Test_Search#test_search_add", "labels": [{"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "test_search"}, {"name": "subSuite", "value": "Test_Search"}, {"name": "host", "value": "hz-zb-zhang"}, {"name": "thread", "value": "11732-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.test_search"}]}